data/data/google/bootstrap/main.tf

```diff
@@ -0,0 +1,59 @@
+resource "google_service_account" "cluster" {
+  account_id   = "${var.cluster_name}-cluster"
+  display_name = "Cluster service account"
+}
+
+resource "google_project_iam_member" "cluster" {
+  role    = "roles/editor"
+  member  = "serviceAccount:${google_service_account.cluster.email}"
+}
+
+resource "google_compute_instance" "bootstrap" {
+  name         = "${var.cluster_name}-bootstrap"
+  machine_type = "${var.instance_type}"
+  zone         = "${var.zone}"
+
+  metadata = {
+    user-data = "${var.ignition}"
+  }
+
+  tags = ["ocp", "ocp-master"]
```

Comment:
I recommend adding the the ability for a user to add additional tags to this list. This will be necessary for environments with restrictive GCP firewalls that require specific tags to be set to allow network communication.